scripts/jenkins/jobs-ibs/cloud-mkcloud7-gating.yaml

```diff
@@ -0,0 +1,56 @@
+- job:
+    name: 'cloud-mkcloud7-gating'
+    project-type: multijob
+    node: cloud-trigger
+    disbaled: true
+
+    triggers:
+      - timed: 'H 19 */2 * *'
```

Comment:
could use 'H H */2 * *' to run at a random hour. or we decrease frequency to weekly because the url-trigger is the more important one.